documentation/ci-fix.md

```diff
@@ -49,18 +49,31 @@ Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/st
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
 ## Breaking Change Check
-
+There are two kind of breaking change checks: same version breaking change check and cross version breaking change check. Both uses the OAD (OpenAPI Diff) tool to identify breaking changes between swaggers.
+- The same version breaking change check compares the existing version swaggers with updated swaggers of the same version, so that to identify if there are any updates break the version.
+- The cross version breaking change check usually applies to new API version scenario. It uses swaggers of the latest public version as the baseline, comparing swaggers of new version with the baseline, so that to identify if there are any updates lead to breaking experience cross versions.
+The latest public version can be:
+1. The latest stable version which released within 3 years.
+2. When there is no stable version released within 3 years, then the latest preview version which released within 1 years.  
+3. The latest public preview version which released over than 1 year.
```

Comment:
I think you can delete #3

Reply:
Here logic is:  if latest preview version is over 1 year. it can not be broken(no matter if there is stable version)
So it should be keep if you are agree above #2 comment . right?

Reply:
More specifically, a preview cannot break customers using the last GA (without approval).
A preview CAN break the last preview.
We do have a policy that a feature should be in preview for > 1 year but I'm not sure tooling should look for this. For example, what if preview-1 of a feature comes out in January and then preview-2 comes out in December. The 1st preview was 11 months old and then the new preview just started but the feature itself is in preview for >1 year. I don't know how your tooling can detect this. 

Maybe it would be useful to have tooling that periodically scans the repo looking for a preview (with no GA) that has been around for 9 months and then notify the team that they have 3 months left to go GA?

Also, perhaps tooling that finds old preview swaggers that do have GA swagger and tell the team to delete them from the repo so as to not clutter the repo with old preview swaggers and to reduce customer confusion when they see all the swaggers in the repo..

Reply:
We just simply use date in version as time. it maybe not very accurate.  yes, it seems a good idea to scan repo, we will discuss it further internally then get back to you.